File: MODUL-1/bahasa c/PRAK0107-231081721007-RaymondHariyono.c
#include <stdio.h>
int main(){

int a , b , c;
   printf("nilai a: ");
   scanf("%d" , & a);
   
   printf("nilai b: ");
   scanf("%d" , & b);
   
   printf("nilai c: ");
   scanf("%d" , & c);
   
float keliling = a + b + c;
int hargaTanah = 85000;
int biayaDiperlukan= keliling * hargaTanah;
   printf("Diketahui : \n");
   printf("Panjang sisi segitiga berturut-turut adalah %d, %d dan %d \n", a,b,c);
   printf("Keliling Tanah Pak Dengklek adalah %.0f \n" , keliling);
   printf("Harga tanah Per Meter adalah %d  \n" , hargaTanah);
   printf("jawaban: \n");
   printf("Biaya yang diperlukan Pak Dengklek adalah : Rp %d",biayaDiperlukan);

return 0;
}

File: MODUL-0/MODUL-1/python/PRAK0107-231081721007-RaymondHariyono.py
a = int(input("nilai a "))
b = int(input("nilai b "))
c = int(input("nilai c "))

keliling = a + b + c
hargaTanah = 85000
biayaDiperlukan = keliling * hargaTanah

print("diketahui : ")
print(f"Panjang sisi segitiga berturut-turut adalah {a}, {b} dan {c}") 
print(f"Keliling Tanah Pak Dengklek adalah {keliling}")
print(f"Harga tanah Per Meter adalah {hargaTanah} ")
print("jawaban : ")
print(f"Biaya yang diperlukan Pak Dengklek adalah : Rp {biayaDiperlukan}")
